_examples/docscansandbox/demo_test.go

```diff
@@ -0,0 +1,338 @@
+			"nationality":     "GBR",
+			"date_of_birth":   "1986-06-01",
+			"document_number": "123456789",
+		}).
+		WithDocumentFilter(documentFilter).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	checkReports, err := request.NewCheckReportsBuilder().
+		WithAsyncReportDelay(5).
+		WithDocumentAuthenticityCheck(documentAuthenticityCheck).
+		WithDocumentTextDataCheck(textDataCheck).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	textExtractionTask, err := task.NewDocumentTextDataExtractionTaskBuilder().
+		WithDocumentFields(map[string]string{
+			"full_name":       "John Doe",
+			"nationality":     "GBR",
+			"date_of_birth":   "1986-06-01",
+			"document_number": "123456789",
+		}).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	taskResults, err := request.NewTaskResultsBuilder().
+		WithDocumentTextDataExtractionTask(textExtractionTask).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	responseConfig, err := request.NewResponseConfigBuilder().
+		WithCheckReports(checkReports).
+		WithTaskResults(taskResults).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	configErr := c.client.ConfigureSessionResponse(c.getIFrameSessionID(), responseConfig)
+
+	if configErr != nil {
+		request, _ := json.Marshal(responseConfig)
+		return errors.New(string(request) + configErr.Error())
+	}
+
+	return configErr
+}
+
+func (c *webContext) iAmOn(path string) error {
+	return c.wd.Get(fmt.Sprintf("%s%s", appBaseUrl, path))
+}
+
+func (c *webContext) clickOn(selector string) error {
+	elem, err := c.wd.FindElement(selenium.ByCSSSelector, selector)
+	if err != nil {
+		panic(err)
+	}
+
+	return elem.Click()
+}
+
+func (c *webContext) iChoose(value string) error {
+	return c.clickOn(fmt.Sprintf("input[value='%s']", value))
+}
+
+func (c *webContext) iClickOnAddPhotoButton() error {
+	return c.clickOn(fmt.Sprintf("*[data-qa='addPhotoButton']"))
+}
+
+func (c *webContext) iClickOnFinishButton() error {
+	return c.clickOn(fmt.Sprintf("*[data-qa='finish-button']"))
+}
+
+func (c *webContext) iShouldSee(text string) error {
+	source, err := c.wd.PageSource()
+	if err != nil {
+		return err
+	}
+	if !strings.Contains(source, text) {
+		return fmt.Errorf("Page source does not contain \"%s\"", text)
+	}
+	return nil
+}
+
+func (c *webContext) iSwitchToTheIframe() error {
+	iFrame, err := c.wd.FindElement(selenium.ByTagName, "iframe")
+	if err != nil {
+		panic(err)
+	}
+	return c.wd.SwitchFrame(iFrame)
+}
+
+func (c *webContext) iUploadADocument() error {
+	uploadElement, err := c.wd.FindElement(selenium.ByCSSSelector, "input[data-qa='change-photo']")
+	if err != nil {
+		panic(err)
+	}
+	return uploadElement.SendKeys(documentImagePath)
+}
+
+func (c *webContext) iWaitSeconds(seconds int) error {
+	time.Sleep(time.Duration(seconds) * time.Second)
+	return nil
+}
+
+func (c *webContext) elementContains(selector string, text string) error {
+	elem, err := c.wd.FindElement(selenium.ByCSSSelector, selector)
+	if err != nil {
+		panic(err)
```

Comment:
Same with the two in this function. Unless there's a reason we're panicking and not returning?

Reply:
Makes sense to return error instead